### PR TITLE
Update explosives to use corrrect explosive power, casing mass, and fragment mass.

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary
```SUMMARY: Balance "Makes explosives use correct values"```

#### Purpose of change
Many explosives had nonsensical (or slightly off) values for their explosive power, casing mass, and fragment mass.
#### Describe the solution
Go over all the bombs and give them correct values.
Explosive power is grams of tnt equivalent.
Casing mass is the weight of all the items that make up the casing, or a value that matches the items weight if some of it is discarded (pipe bomb).
Fragment mass is the weight of a nail for items that use nails as fragments (0.8g), and higher than that for explosives with no method of causing extra fragmentation.

#### Describe alternatives you've considered
Wait for consensus on #37705 and leave the values all wacky until then.

#### Testing
Game loads and items have the correct values.

### CODE REVIEW

**Comment by Brian-Otten** [comment]
Adjusted so bombs with nails have fragments the size of half a nail (0.4g), and bombs were the casing fragments have 1g fragments.

**Comment by I-am-Erk** [comment]
It's tough to review this. Could you summarize the rationale behind the changes you picked?

**Comment by Brian-Otten** [comment]
For explosive power i used the grams of black powder used in the recipe, x 0.55, to get tnt equivalence. Some of the other ingredients are not correctly proportioned in comparison to black powder for tnt equivalence, but that is being fixed in other PR's i have open currently. I rounded down slightly for round numbers to match the other explosives in game.

So for the black powder charge, it takes 4120 black powder, which is 2266 grams of tnt equivalent, which i rounded to 2250 explosive power.

For the casing on the black powder charge, we see that 20 scrap metal is included, with a weight of 2kg, since that is going to make up the fragment released from the blast we get a casing mass of 2000g.

The main thing that is up for discussion is the fragment size. As mentioned above, i'm now using half a nail as the fragment size for bombs that use nails as their fragments (0.4g), and 1g fragments for bombs with larger metal objects being fragmented by the blast themselves.

If you want to audit that i got it right, go through the ones i changed and compare explosive power to black powder used in the recipe, and compare casing mass to the weight of the metal ingredients used.
